Add SBLYR to registry

keyflight opened the pull request.

lophre commented on registry.json:
```diff
@@ -1997,5 +1997,20 @@
+      "website": "https://pool.sublayer.io",
+      "youtube": "https://www.youtube.com/channel/UC5MjFaqqlAbTX5NtxbG2_6Q"
+    },
+    "telegram": "sblyr",
```
You need to provide a Telegram username.  A group name is not accepted.  The SPA uses Telegram to gather votes from it's members and we use the username to let our members in the voting group.

keyflight replied:
personal telegram handle is sublayerio

lophre commented on registry.json:
```diff
@@ -1997,5 +1997,20 @@
+      "youtube": "https://www.youtube.com/channel/UC5MjFaqqlAbTX5NtxbG2_6Q"
+    },
+    "telegram": "sblyr",
+    "operatorName": "sublayerio",
```
Leave this field empty if you don't provide a real first name or full name.

keyflight replied:
firstname is Olivier

lophre reviewed (changes requested):
Please see comments.

keyflight commented:
@lophre thanks for reviewing. i've adjusted the registry.json, i've also tried pulling the latest registry.json and added my pool in there so you would perhaps be able to merge it at once.

leo42 commented:
Hey SBLYR, 

Please close your pool request and create from clean.

keyflight commented:
@leo42 no offence, there's quite some room for improvement in this process. I've first created a PR 3 weeks ago.

keyflight commented:
@leo42 i prepared it for you; https://github.com/SinglePoolAlliance/Registration/pull/242
